cmd/backup-manager/app/clean/clean.go

```diff
@@ -128,9 +130,43 @@ func (bo *Options) deleteVolumeSnapshots(meta *bkutil.EBSBasedBRMeta) error {
 		klog.Errorf("new a ec2 session failure.")
 		return err
 	}
-	if err = ec2Session.DeleteSnapshots(newVolumeIDMap); err != nil {
-		klog.Errorf("delete snapshot failure.")
-		return err
+
+	// https://docs.aws.amazon.com/AWSEC2/latest/APIReference/throttling.html DeleteSnapshot fill rate is 5 per second
+	// Each retry interval is 2 second, and no more than 100 times retry (~3 minutes)
+	for {
+		backoff := wait.Backoff{
```

Comment:
I think you should retry in the worker pool of `ec2Session.DeleteSnapshots` method. The error returned from `ec2Session.DeleteSnapshots` is a `multierr`, not original `awserr`

Reply:
I would like to keep the retry outside.